### PR TITLE
Silent updates for comments

### PR DESCRIPTION
**Pull request by pleonex**
After #214 we are able to make silent changes in Jira issues. However, the update method for comment doesn't allow to specify the notify parameter. This PR adds this argument for comments too.

### CODE REVIEW

**Comment by pleonex** [comment]
The CI failure is not related to my changes. It seems to be an error related to the ChangeLog file. The develop branch is broken too:
https://travis-ci.org/pycontribs/jira/jobs/259025381

File: jira/resources.py [hunk 1]
```diff
@@ -516,13 +516,13 @@ def __init__(self, options, session, raw=None):
         if raw:
             self._parse_raw(raw)
 
-    def update(self, fields=None, async=None, jira=None, body='', visibility=None):
+    def update(self, fields=None, async=None, jira=None, body='', visibility=None, notify=True):
```
Comment (hdost) [inline jira/resources.py]: Keep this

File: jira/resources.py [hunk 2]
```diff
@@ -516,13 +516,13 @@ def __init__(self, options, session, raw=None):
...
         data = {}
         if body:
             data['body'] = body
         if visibility:
             data['visibility'] = visibility
-        super(Comment, self).update(data)
+        super(Comment, self).update(data, notify=notify)
```
Comment (hdost) [inline jira/resources.py]: Keep this

File: tests/tests.py [hunk 3]
```diff
@@ -1062,6 +1062,12 @@ def test_update_comment(self):
         # self.assertEqual(comment.visibility.value, 'Administrators')
         comment.delete()
 
+    def test_update_comment_with_notify_false(self):
```
Comment (hdost) [inline tests/tests.py]: Keep this

**Comment by hdost** [comment]
Feel free to rebase of master, and heed the comments above, and submit a new PR.
Closing this PR.

**Pull request closed without merging**